build-system/eslint-rules/anchor-mutator-priority.js

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
No need for a lint rule if we use the `dev().assert`.

Reply:
changed to dev assert and reverted lint rule. Thanks.